app/lib/bk/compat/parsers/bitrise.rb

```diff
@@ -41,12 +42,22 @@ def parse
 
         stack_value = @config.dig('meta', 'bitrise.io', 'stack')
         agents = stack_value ? { stack: stack_value } : {}
+
         Pipeline.new(
           steps: bk_steps,
-          agents: agents
+          agents: agents,
+          env: process_app_envs(@config.dig('app', 'envs') || {})
         )
       end
 
+      def process_app_envs(vars)
+        return {} unless vars # takes care of empty vars and nils
+
+        result = {}
+        BK::Compat::Bitrise::EnvHelper.process_env_variables(vars, result)
+        result
+      end
+
```

Comment:
This should just be a call to `BK::Compat::Bitrise::EnvHelper.process_env_variables` that just returns the corresponding dict